src/controllers/user/other.rs

```diff
@@ -29,11 +30,13 @@ pub async fn stats(state: AppState, Path(user_id): Path<i32>) -> AppResult<Json<
 
         let conn = &mut *state.db_read_prefer_primary()?;
 
-        let data: i64 = CrateOwner::by_owner_kind(OwnerKind::User)
+        let data = CrateOwner::by_owner_kind(OwnerKind::User)
             .inner_join(crates::table)
+            .inner_join(crate_downloads::table.on(crates::id.eq(crate_downloads::crate_id)))
             .filter(crate_owners::owner_id.eq(user_id))
-            .select(sum(crates::downloads))
-            .first::<Option<i64>>(conn)?
+            .select(sum(crate_downloads::downloads))
+            .first::<Option<BigDecimal>>(conn)?
+            .map(|d| d.to_u64().unwrap_or(u64::MAX))
             .unwrap_or(0);
```

Comment:
Just leaving information here. Due to the fact that the `sum` function over `bigint` columns in PostgreSQL returns `numeric` (`sum ( bigint ) → numeric`), I believe this is the reason we use `BigDecimal` here instead of `i64`.

Reply:
yeah, exactly. Diesel has the same behavior built into the typesystem, which is why I had to change it like this